src/components/layout/nav/nav-interim.tsx

```diff
@@ -46,42 +46,57 @@ const RightSection = styled(Flex)`
     }
 `
 
-const NavInterim = ({ interim_type, landing_type }: NavInterimProps) => (
-    <NavTemplate nav_height="8.6rem">
-        <Container jc="space-between" p="2.4rem 0">
-            <Flex ai="center" jc="flex-start">
-                <DesktopWrapper>
-                    <LogoWrapper to={`/interim/${interim_type}/`} aria-label="Home">
-                        <Flex ai="center">
-                            <img src={Logo} alt="deriv logo" width="190" height="27" />
-                            <img src={GetTrading} alt="get trading" width="120" height="17" />
-                        </Flex>
-                    </LogoWrapper>
-                </DesktopWrapper>
-
-                <MobileWrapper>
-                    <LogoLinkMobile to={`/interim/${interim_type}/`} aria-label="Home">
-                        <Flex>
-                            <img src={LogoOnly} alt="deriv logo" width="115" height="27" />
-                            <LogoDescription ai="center">
-                                <Line />
-                                <img src={GetTrading} alt="get trading" width="120" height="17" />
-                            </LogoDescription>
-                        </Flex>
-                    </LogoLinkMobile>
-                </MobileWrapper>
-            </Flex>
-
-            <RightSection jc="flex-end" ai="center">
-                {!landing_type && <LanguageSwitcher has_short_name />}
-                {!landing_type && (
-                    <StyledLinkButton secondary to="/">
-                        {localize('Explore Deriv.com')}
-                    </StyledLinkButton>
-                )}
-            </RightSection>
-        </Container>
-    </NavTemplate>
-)
+const NavInterim = ({ interim_type, landing_type }: NavInterimProps) => {
+    const { hide_branding } = React.useContext(DerivStore)
+    return (
+        <NavTemplate nav_height="8.6rem">
+            <Container jc="space-between" p="2.4rem 0">
+                <Flex ai="center" jc="flex-start">
+                    {!hide_branding && (
+                        <DesktopWrapper>
+                            <LogoWrapper to={`/interim/${interim_type}/`} aria-label="Home">
+                                <Flex ai="center">
+                                    <img src={Logo} alt="deriv logo" width="190" height="27" />
+                                    <img
+                                        src={GetTrading}
+                                        alt="get trading"
+                                        width="120"
+                                        height="17"
+                                    />
+                                </Flex>
+                            </LogoWrapper>
+                        </DesktopWrapper>
+                    )}
+                    {!hide_branding && (
```

Comment:
```suggestion
```
You don't need this.

Reply:
i tried doing it before by removing it but it throws error.so wrapped separately

Reply:
You need a parent container `<></>` to make this work

Reply:
okay i will add